lib/generators/provider/templates/.travis.yml

```diff
@@ -9,6 +9,7 @@ env:
   - RUBY_GC_HEAP_GROWTH_MAX_SLOTS=300000
   - RUBY_GC_HEAP_INIT_SLOTS=600000
   - RUBY_GC_HEAP_GROWTH_FACTOR=1.25
+bundler_args: --no-deployment
```

Comment:
@NickLaMuro 

>  bundling twice

can you point me to where this happens?
Maybe this does not happen anymore with the changes in this PR?

See the travis log for https://github.com/ManageIQ/manageiq-providers-ansible_tower/pull/11
I dont see running bundle install twice there